phable_cli/cli.py

```diff
@@ -201,15 +205,20 @@ def move_task(ctx, task_ids: list[int], column: str | None):
 
     """
     client = PhabricatorClient()
-    if not (
-        current_milestone := client.get_project_current_milestone(
+    if milestone:
+        if not (
+            current_milestone := client.get_project_current_milestone(
+                project_phid=os.environ["PHABRICATOR_DEFAULT_PROJECT_PHID"]
+            )
+        ):
+            ctx.fail("Current milestone not found")
+        current_milestone_columns = client.list_project_columns(
+            project_phid=current_milestone["fields"]["proxyPHID"]
+        )
+    else:
+        current_milestone_columns = client.list_project_columns(
```

Comment:
```suggestion
        board_columns = client.list_project_columns(
            project_phid=current_milestone["fields"]["proxyPHID"]
        )
    else:
        board_columns = client.list_project_columns(
```

I think it's cleaner to rename that variable now that we can move tasks _outside_ of the context of a milestone.

Reply:
I choose instead to extract some of that logic to PhabricatorClient. It seems more readable to me (and also more reusable, choosing milestone / main project looks like a common concern).

Note that I'm not really sure how to use Python's exceptions, so that part might be more messy than it needs to be.